webui/react/src/pages/TrialDetails/TrialInfoBox.tsx

```diff
@@ -50,6 +53,33 @@ const TrialInfoBox: React.FC<Props> = ({ trial, experiment }: Props) => {
         return `${trial?.logRetentionDays} days`;
     }
   }, [trial]);
+  const integrationData = useMemo(() => {
+    const {
+      config: { integrations },
+    } = experiment;
+
+    if (integrations) {
+      const url = createPachydermLineageLink(integrations);
+
+      return {
+        hasIntegrationData: url !== undefined ? true : false,
+        text: '<MLDM repo>',
+        url,
+      };
+    }
```

Comment:
as mentioned previously, an empty object is a valid integrations config. we shouldn't show the repo card if the user can't click on it.